docusaurus.config.js

```diff
@@ -155,6 +155,11 @@ const config = {
             position: "left",
             target: "_self",
           },
+          {
+            label: "Supporters",
+            href: "/#supporters",
+            position: "left",
+          },
```

Comment:
```suggestion
```
No need to add to nav bar  :)

Reply:
> 

Ok, I thought it might be helpful, anyway I removed that!